approval_polls/tests.py

```diff
@@ -161,6 +161,31 @@ def test_vote_view_counts_increase(self):
         self.assertContains(response, '21 votes')
         self.assertContains(response, '101 ballots', status_code=200)
 
+class MyPollTests(TestCase):
+    def setUp(self):
+        self.client = Client()
+        create_poll(question="question1", days=-5)
+        create_poll(question="question2", username="user2", days=-5)
+
+    def test_redirect_when_not_logged_in(self):
+        """
+        If the user is not logged in then redirect to the login page
+        """
+        response = self.client.get(reverse('login'))
+        self.assertEqual(response.status_code, 200)
```

Comment:
This test is just checking if hitting the login url returns a 200. It should be checking if hitting the approval_polls:my_polls url returns a 302 redirect to the login page.